modules/microshift-restart-ovnkube-master.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-networking.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-restart-ovnkube-master_{context}"]
+= Restarting the ovnkube-master pod
+
+The following procedure restarts the ovnkube-master pod.
+
+.Prerequisites
+
+* The OpenShift CLI (`oc`) is installed.
+* Access to the cluster as a user with the `cluster-admin` role.
+* A cluster installed on infrastructure configured with the OVN-Kubernetes network plugin.
+* The KUBECONFIG environment variable is set.
+
+.Procedure
+
+Run the commands listed in each step that follows to restart ovnkube-master pod.
+
+. Access the remote cluster by running the following command:
++
+[source, terminal]
+----
+$ export KUBECONFIG=$PWD/kubeconfig
+----
+
+. Find the name of the ovnkube-master pod that you want to restart by running the following command:
++
+[source, terminal]
+----
+$ pod=$(oc get pods -n openshift-ovn-kubernetes | awk -F " " '/ovnkube-master/{print $1}')
+----
+
+. Delete the ovnkube-master pod by running the following command:
++
+[source, terminal]
+----
+$ oc -n openshift-ovn-kubernetes delete pod $pod
+----
+
+. Confirm that a new ovnkube-master pod has been started by running the following command:
++
+[source, terminal]
+----
+$ oc get pods -n openshift-ovn-kubernetes
+----
+The listing of the running pods shows a new ovnkube-master pod name and age.
```

Comment:
This sentence seems out of place. Could this be a verification step maybe?

Reply:
What we really need here is an example of the output when new pods launch. I haven't had time to add that.